include/uk/arch/paging.h

```diff
@@ -370,14 +370,14 @@ int PAGE_Lx_IS(__pte_t pte, unsigned int lvl);
  * 64 bits for the virtual address.
  *
  * @param start the start of the virtual address range
- * @param end the end of the virtual address range
- *
+ * @param  len the length of the virtual address range, where the end address is
+ *	(start + len - 1)
```

Comment:
The ", where..." part is unnecessary IMO

Reply:
OK I'll remove the part